themes/docs/templates/Includes/DocumentationFooter.ss

```diff
@@ -1,5 +1,5 @@
 <div id="footer">
-	<p>Documentation powered by <a href="http://www.silverstripe.org">SilverStripe</a> (<a href="https://github.com/silverstripe/doc.silverstripe.org">code</a>). <a href="https://github.com/silverstripe/silverstripe-framework">Contribute documentation to doc.silverstripe.org</a>, <a href="https://github.com/silverstripe/silverstripe-framework/issues/new?labels=documentation">raise a bug or enhancement ticket</a>. 
+	<p>Documentation powered by <a href="http://www.silverstripe.org">SilverStripe</a> (<a href="https://github.com/silverstripe/doc.silverstripe.org">code</a>). <a href="https://github.com/silverstripe/silverstripe-framework">Contribute documentation to doc.silverstripe.org</a>, <a href="https://github.com/silverstripe/silverstripe-framework/issues/new?labels=documentation">raise a bug or enhancement ticket</a>.</p>
 
-	<a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a><br />This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>.
+	<p><a rel="license" href="http://creativecommons.org/licenses/by/4.0/" ><img alt="Creative Commons License" style="border: 0; float: right; margin-left: 6px;" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a>This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>.</p>
```

Comment:
Can we please put CSS styles into the stylesheet, rather than inline style attributes?

Reply:
Sure